tests/Samples.Complex.Tests/BaseTest.cs

```diff
@@ -15,6 +16,14 @@ protected BaseTest()
     {
         Host = Program.CreateHostBuilder(Array.Empty<string>())
              .ConfigureWebHost(webBuilder => webBuilder.UseTestServer())
+             .ConfigureServices(services =>
+             {
+                 services.AddGraphQL(b => b.ConfigureExecution((options, next) =>
```

Comment:
How did it work before without `AddGraphQL`?

Reply:
It runs `AddGraphQL` within the `Program.cs` file.  For some reason the change to `UseApolloTracing` now started returning tracing results in the tests (which it should, I believe).  So this call to `AddGraphQL` serves only to disable metrics for the purposes of the tests, leaving the sample project in tact.

Reply:
Oh, the reason is obvious -- the call to ConfigureExecutionOptions sets EnableMetrics, which now occurs prior to `UseApolloTracing`.  I should have changed `UseApolloTracing()` to `UseApolloTracing(false)`, as the sample is now incorrect.

Reply:
#903